### ISSUE TITLE
Underline does not span spaces.

### ISSUE THREAD

**Issue by renpytom** [renpy/renpy#1605]
https://lemmasoft.renai.us/forums/viewtopic.php?p=498135#p498135

**Comment by renpytom** [comment]
I'd recommend against it. I was planning on taking a stab tonight. I'm
about 90% sure it's something very simple I did. but I know where I work so
it'll be easy enough for me to do it.

**Comment by ShehriyarQureshi** [comment]
Yup I was going through the files and it seems it shouldn't be hard. Alright you can do it :smile:
